### ISSUE TITLE
Mount folders outside of the project root

### ISSUE THREAD

**Issue by drupov** [lando/lando#753]
I am using a very basic lando install for a Drupal 7 site and have an additional styleguide that is providing styling.

So far I was able to develop those styles separately in a own environment (with all the node_modules needed installed there) and Drupal was only referencing/pointing to the generated styles from the styleguide.

With lando I cannot use that approach anymore, as the app cannot find those styleguide files, as they are outside of the app folder (or subfolders), e.g.

```
/home/user/apps/drupal7 # <- this is where the .lando.yml file is located
/home/user/apps/drupal7/web # <- the root of the Drupal installation
/home/user/apps/styleguide # <- the root of the the styleguide, which is not included in the lando containers
```

Is there a way to specify in my .lando.yml file that I want to mount the `/home/user/apps/styleguide` folder, so that is included in the mounted lando containers and therefore accessible from there?

Thanks

**Comment by pirog** [comment]
@drupov the lando approach is to have EVERYTHING you need for a given project inside of the same directory. 

That said it sounds like your styleguide is actually its own separate project, in this case you'd probably want to give it its own landofile and have both apps turned on.

**Comment by drupov** [comment]
@pirog thanks, that makes sense. Maybe I did not explain it that well, but the styleguide runs separately anyway. I just need to import its' generated styles into the drupal installation which runs in lando.

The only way I see that happening would be to mount the folder, where the styleguide is placed as a folder in the lando containers. From there it's easy to point the drupal theme to the location of the generated css file from the styleguide.

Anyway, after some research I believe that is achievable by adding the following into the .lando.yml file:
```
services:
  appserver:
    overrides:
       services:
         volumes:
           - /path/to/styleguide/on/local:/app/styleguide
```

**Comment by pirog** [comment]
You can definitely do that but seems very brittle eg it breaks the core
lando idea which is you can have another user clone the repo and run lando
start and get rolling. This couples two things together in a non lando y
way. I'd probably do something like `run styleguide build elsewhere -> add
build artifact (eg css/js) to the drupal site and check it into version
control -> repeat when needed.

Obviously it's your site so you do you but something like that would be my
high level recommendation :)

**Comment by danyg79** [comment]
I looked for the very similar things, i'm in the middle of two paralell development: there is a Drupal 7 site what has active development threads, and a request to upgrade the site to Drupal 9. The files directory of this site is more than 50GB, I didn't want to duplicate it on my local machine, so I had to share the files directory for both sites. 

Now, my D7 site's config contains the following and works properly:
`services:
  appserver:
    overrides:
      volumes:
        - /path/to/shared/files:/app/web/sites/default/files
`
Thank you @drupov (I haven't heard about you for years) for pointing me to the right direction!
